app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Please try to keep unrelated changes out of the commit

Reply:
Sure! This was me trying make the Firebase work without androidx. Sorry about that.